examples/hello-world/hello-flower/flwr-pt-tb/pyproject.toml

```diff
@@ -2,6 +2,13 @@
 requires = ["hatchling"]
 build-backend = "hatchling.build"
 
+# Tested with:
+#   flwr==1.16.0
+#   nvflare==2.6.0
+#   torch==2.7.0
+#   torchvision==0.22.0
```

Comment:
the torchvision version is also different from the original.

Reply:
I run this locally with new versions